### DISCUSSION TITLE
Suppress display area

### DISCUSSION THREAD

**Discussion by borat99Z** [uBlockOrigin/uBlock-issues#2451] [Q&A]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

I have been a uBlock user for several years and I have already created many filters, but the filter to not display a certain area :
       `www.lequipe.fr##div.Home__widget.CarouselWidget:nth-of-type(2)`
on the home page of the site
       `https://www.lequipe.fr/`
does not work at all !

Many Thanks !

### A specific URL where the issue occurs.

 https://www.lequipe.fr/

### Steps to Reproduce

 https://www.lequipe.fr/
apply filter 

### Expected behavior

Suppress the displayed area

### Actual behavior

Area is still displayed

### Configuration

uBlock Origin 1.46.0
Firefox 102.0

**Comment by gothic-bum** [comment]
I just tested and the filter `www.lequipe.fr##div.Home__widget.CarouselWidget:nth-of-type(2)` works properly. Have you used the logger to determine if you have any exception filters being applied?

And since this isn't an issue with uBO itself, you should post questions like this to the support forum at: [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).

**Reply by krystian3w** [reply]
workaround based on fake element:

```adb
lequipe.fr##div.Home__widget.CarouselWidget:nth-of-type(2), badfilter
```

then filter tries hide two elements if first is in `#@#` syntax.

Don't compatible with `shide`/`ehide` enabled.
